ScrollableTabBar.js

```diff
@@ -136,14 +135,24 @@ const ScrollableTabBar = React.createClass({
       onPress={() => this.props.goToPage(page)}
       onLayout={this.measureTab.bind(this, page)}
     >
-      <View style={[styles.tab, this.props.tabStyle, ]}>
-        <Text style={[{color: textColor, fontWeight, }, textStyle, ]}>
-          {name}
-        </Text>
-      </View>
+      {this.renderTabName(name, page, isTabActive)}
     </Button>;
   },
 
+  renderTabName(name, page, isTabActive) {
+    const { activeTextColor, inactiveTextColor, textStyle, } = this.props;
+    const textColor = isTabActive ? activeTextColor : inactiveTextColor;
+    const fontWeight = isTabActive ? 'bold' : 'normal';
+
+    return (
```

Comment:
last nitpick. I'm trying to follow khan academy style guide
https://github.com/Khan/style-guides/blob/master/style/react.md#open-elements-on-the-same-line
Could you please fix return statement